auditlog/models/log.py

```diff
@@ -18,7 +18,6 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-
```

Comment:
I agree, but I would like to do these kind of changes in another PR (simplify headers, add screenshots, improve/clean up the README...).